packages/node/src/plugins/NodeFederationPlugin.ts

```diff
@@ -24,6 +24,10 @@ interface Context {
 
 
 export const parseRemotes = (remotes: Record<string, any>) => Object.entries(remotes).reduce((acc, remote) => {
+  if(remote[1].startsWith('internal ')) {
```

Comment:
skip again if internal modules, will add utils so its easier to implement without boilerplate